server_plugin/server.py

```diff
@@ -194,6 +206,9 @@ def start(vca_client, **kwargs):
 @operation
 @with_vca_client
 def stop(vca_client, **kwargs):
+    if ctx.node.properties.get('use_external_resource'):
```

Comment:
minor nitpick, i apologize in advance: this style of returning if it is in fact use_external_resource is contradictory to the opposite which is used in the "start" method. personally i'd use the one here for both; i think it'd be a bit more readable if we were to be consistent about it in any case.